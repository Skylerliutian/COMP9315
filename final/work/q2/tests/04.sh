./bnl 3 6 2
