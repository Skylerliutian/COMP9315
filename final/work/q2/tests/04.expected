Running: ./bnl 3 6 2

Frames:     [00] [01]
Contents:    R02  S05
PinCount:      0    0
Popularity:    1    2
Clock: 0

#requests: 21
#releases: 21
#hits    : 0
#reads   : 21
